sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -24,6 +24,7 @@
 #include "../common/soc-intel-quirks.h"
 #include "hda_dsp_common.h"
 #include "sof_maxim_common.h"
+#include "../../sof/sof-audio.h"
```

Comment:
@keyonjie This pops out a bit. This would seem like a pretty generic need (for any machine driver), so I wonder if we could find a cleaner way. At least make it an official API machine drivers can use.

Reply:
@kv2019i makes sense, as @lgirdwood commented above, let me change it like that.